cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala

```diff
@@ -339,6 +341,7 @@ All users may view all of their own currently executing transactions.
 |a|The result of the `TERMINATE TRANSACTION` command as applied to this transaction.
 |m|STRING
 ||===""")
+      p("The `TERMINATE TRANSACTIONS` command can be combined with multiple `SHOW TRANSACTIONS` and `TERMINATE TRANSACTIONS`, see <<query-combine-tx-commands, transaction commands combination>>.")
```

Comment:
```suggestion
      p("The `TERMINATE TRANSACTIONS` command can be combined with multiple `SHOW TRANSACTIONS` and `TERMINATE TRANSACTIONS`.
      See <<query-combine-tx-commands, transaction commands combination>>.")
```

Reply:
same here

Reply:
What if no new line is added here, but we keep the full stop? 

```suggestion
      p("The `TERMINATE TRANSACTIONS` command can be combined with multiple `SHOW TRANSACTIONS` and `TERMINATE TRANSACTIONS`. See <<query-combine-tx-commands, transaction commands combination>>.")
```